infra/cifuzz/cifuzz.py

```diff
@@ -410,8 +410,9 @@ def get_json_from_url(url):
     logging.error('HTTP error with url %s.', url)
     return None
   try:
-    result_json = json.load(response)
-  except ValueError as excp:
+    # read().decode() fixes compatability issue with urllib response object.
+    result_json = json.loads(response.read().decode())
```

Comment:
super nit: when possible, it's nice to write more descriptive commit message, e.g. `[CIFuzz] Fix JSON loading in affected fuzzers feature.`